scripts/soroban-settings/SorobanSettingsUpgrade.py

```diff
@@ -31,25 +31,85 @@
 
 # hardcode the upgrade you want to do here.
 def get_upgrade_set():
```

Comment:
General comment: it would be very helpful to add `_` delimiters to all the numbers to make this more readable.